.github/issue-labeler-config/regexes-for-issue-labels.yaml

```diff
@@ -0,0 +1,43 @@
+# SPDX-FileCopyrightText: 2009 Fermi Research Alliance, LLC
```

Comment:
For this name I meant directly `.github/regexes-for-issue-labels.yaml` or `.github/issue-labeler-config-regexes.yaml`. No need to add one folder for a single file.

Reply:
updated to be directly located under the top-level directory